src/components/Pega_Extensions_SecureRichText/utils.tsx

```diff
@@ -0,0 +1,78 @@
+import { EmailDisplay, PhoneDisplay } from '@pega/cosmos-react-core';
+
+type ActionType = (propName: string, value: string) => void;
+
+export const handleEvent = (
```

Comment:
if the onImageAdded is removed - this function is only used for ChangeNBlur - in this case, we should remove and move the 2 lines of code in the index